### PR TITLE
Improve javadoc for all addons

### PR DESCRIPTION
**Pull request by holgerfriedrich**
The javadoc was quite broken form most of the addons.

This PR fixes most of the errors the javadoc run is reporting.

Sorry for bothering almost everybody with this review request.

### CODE REVIEW

**Review by clinique — APPROVED** [review]
LGTM

**Review by robnielsen — APPROVED** [review]

**Review by mlobstein — APPROVED** [review]
lgtm

**Review by mherwege — APPROVED** [review]
LGTM

**Review by marcelrv — APPROVED** [review]
LGTM for max, miio, windcentrale,,ntp bindings

**Review by mvalla — APPROVED** [review]
LGTM for openwebnet

File: bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/api/MessageReceivedEvent.java [hunk 1]
```diff
@@ -16,7 +16,8 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 /**
- * The {@link DataPointChangedEvent} is an event container for data point changes
+ * The {@link org.openhab.binding.ism8.server.DataPointChangedEvent}
+ * is an event container for data point changes
```
Comment (hmerk) [inline bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/api/MessageReceivedEvent.java:20]: Please undo this change, you added a comment from the wrong binding (ism8)

**Review by hmerk — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/api/MessageReceivedEvent.java (hunk 1 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/api/MessageReceivedEvent.java:20]: Ok, the proposed change is wrong. But it seems it has already been wrong before the change.
Shouldn't it be ```MessageReceivedEvent````?

**Review by weymann — APPROVED** [review]
Fine for e3dc, mybmw, mercedesme, sensorcommunity

**Review by wborn — APPROVED** [review]
Thanks for the huge cleanup! :+1:

**Review by lsiepel — APPROVED** [review]
LGTM. Did not check all 600+ files, but random looked at 100 or so.

**Review by jlaur — CHANGES_REQUESTED** [review]
Only a minor comment from my side. Thanks for the cleanup, and @wborn, thanks for reviewing.

File: bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java [hunk 2]
```diff
@@ -130,7 +126,6 @@ public static List<SonosEntry> getEntriesFromString(String xml) {
      *
      * @param xml
      * @return The value of the desc xml tag
-     * @throws SAXException
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java:133]: SAXException is there in the method definition.

**Comment by lolodomo** [comment]
Except sonos binding, changes in astro, freebox, freeboxos, lgwebos, netatmo, ntp, powermax, pulseaudio, sonyprojector, weatherunderground looks fine to me. I have not checked the changes in other bindings.

**Review by janvyb — APPROVED** [review]

**Review by GiviMAD — APPROVED** [review]
LGTM for broadlinkthermostat, unifiedremote and googlestt bundles. Thank you!
